client/src/main.ts

```diff
@@ -72,6 +73,16 @@ async function startVue () {
   Vue.prototype.$USAStateCodes = USAStateCodes
   Vue.prototype.$xproMapping = mapping.xproMapping
 
+  // Setup Hotjar
+  // Initialize Launch Darkly
+  if (window['hotjarId']) {
+    console.info('Initializing Hotjar...') // eslint-disable-line no-console
+    Vue.use(Hotjar, {
+      id: window['hotjarId'],
+      isProduction: true
```

Comment:
Should this be changed for non-production? Maybe add something like this:
isProduction: (process.env.NODE_ENV === 'production')

Reply:
We want to open this option in dev and test. Otherwise, we can get the data from hotjar.

isProduction:
If you would like to disable or enable tracking, pass in either true or false. It is advised to bind your Node ENV variable. This is an optional parameter and will default to true if not specified.

Reply:
OK, so we this to run in all environments.

I suggest you delete this property, since it looks like an error, and it defaults to True anyway.

Reply:
Removed as your recommend.